pkg/auth/identityRequest.go

```diff
@@ -18,14 +18,17 @@ type ServiceAccountIdentityRequest struct {
 // CertificateIdentityRequest is the request for a new certificate validation.
 type CertificateIdentityRequest struct {
 	ClusterID                 string `json:"clusterID"`
+	OriginClusterToken        string `json:"originClusterToken,omitempty"`
```

Comment:
Would suggest to add a comment to clarify why this is propagated, and change `Token` below to `DestinationClusterToken` for consistency.